burger-app/src/components/Header/Menu/MenuItem/menuItem.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+const MenuItem = ({ children }) => {
+  return <MenuItemStyled>{children}</MenuItemStyled>;
+};
+
+const MenuItemStyled = styled.li({
```

Comment:
Looks good, only thing that should be fixed is jumping of the element on hover


https://user-images.githubusercontent.com/84010501/208152303-882e7d3c-4f4a-46af-9ae2-e40bc3d71d66.mov